classes/Tools.php

```diff
@@ -4264,10 +4264,11 @@ public static function getPath($url_base, $id_category, $path = '', $highlight =
                 $n = 1;
                 $n_categories = (int) count($categories);
                 foreach ($categories as $category) {
-                    $link = Context::getContext()->link->getAdminLink('AdminCategories');
-                    $edit = '<a href="' . Tools::safeOutput($link . '&id_category=' . (int) $category['id_category'] . '&' . (($category['id_category'] == 1 || $home) ? 'viewcategory' : 'updatecategory')) . '" title="' . ($category['id_category'] == Category::getRootCategory()->id_category ? 'Home' : 'Modify') . '"><i class="icon-' . (($category['id_category'] == Category::getRootCategory()->id_category || $home) ? 'home' : 'pencil') . '"></i></a> ';
+                    $action = (($category['id_category'] == (int) Configuration::get('PS_HOME_CATEGORY') || $home) ? 'admin_categories_index' : 'admin_categories_edit');
+                    $link = Context::getContext()->link->getAdminLink('AdminCategories',true, array('route'=>$action,'categoryId'=>(int) $category['id_category']));
```

Comment:
Sorry I missed this one, last modif you should use the legacy parameter `id_category` and let the converter rename it into `categoryId` for symfony routing

This is one of the advantages I mentioned, you can always inject the legacy parameter and if we, someday, change the parameter name in the routing it will still work.

Reply:
Done !
Thank you @jolelievre

Reply:
Thank YOU! I just approved the PR!